src/transport/tests/TestSecureSessionTable.cpp

```diff
@@ -0,0 +1,424 @@
+    }
+
+    //
+    // This validates evicting an over-minima fabric from the session table where there
+    // are sessions from two fabrics, Fabric1 and Fabric2.
+    //
+    // Fabric1 has 2 sessions, and Fabric2 has 4 sessions. Fabric2 will be selected since
+    // it has more sessions than Fabric2.
+    //
+    // Within that set, there are more sessions to Node 2 than others, so the oldest one
+    // in that set (timestamp 3) will be selected.
+    //
+    {
+        ChipLogProgress(SecureChannel, "-------- Over-minima Fabric Eviction ---------");
+
+        std::vector<SessionParameters> sessionParamList = {
+            { { 2, kFabric1 }, System::Clock::Timestamp(9), SecureSession::State::kActive },
+            { { 2, kFabric2 }, System::Clock::Timestamp(3), SecureSession::State::kActive },
+            { { 1, kFabric1 }, System::Clock::Timestamp(2), SecureSession::State::kActive },
+            { { 2, kFabric2 }, System::Clock::Timestamp(7), SecureSession::State::kActive },
+            { { 3, kFabric2 }, System::Clock::Timestamp(1), SecureSession::State::kActive },
+            { { 4, kFabric2 }, System::Clock::Timestamp(2), SecureSession::State::kActive },
+        };
+
+        _this->CreateSessionTable(sessionParamList);
+        _this->AllocateSession(ScopedNodeId(2, kFabric1), sessionParamList, 1);
+    }
+
+    //
+    // This validates evicting an over-minima fabric from the session table where there
+    // are sessions from two fabrics, Fabric1 and Fabric2.
+    //
+    // Fabric1 has 2 sessions, and Fabric2 has 3 sessions. Fabric2 will be selected since
+    // it has more sessions than Fabric2.
+    //
+    // Within that set, there are more sessions to Node 2 than others, except one session
+    // is in the pairing state. So the active one will be selected instead.
+    //
+    {
+        ChipLogProgress(SecureChannel, "-------- Over-minima Fabric Eviction (State) ---------");
+
+        std::vector<SessionParameters> sessionParamList = {
+            { { 2, kFabric1 }, System::Clock::Timestamp(9), SecureSession::State::kActive },
+            { { 2, kFabric2 }, System::Clock::Timestamp(3), SecureSession::State::kEstablishing },
+            { { 1, kFabric1 }, System::Clock::Timestamp(2), SecureSession::State::kActive },
+            { { 2, kFabric2 }, System::Clock::Timestamp(7), SecureSession::State::kActive },
+            { { 3, kFabric2 }, System::Clock::Timestamp(1), SecureSession::State::kActive },
+            { { 4, kFabric2 }, System::Clock::Timestamp(2), SecureSession::State::kActive },
+        };
+
+        _this->CreateSessionTable(sessionParamList);
+        _this->AllocateSession(ScopedNodeId(2, kFabric1), sessionParamList, 3);
+    }
+
+    //
+    // This validates evicting an over-minima fabric from the session table where there
+    // are sessions from two fabrics, Fabric1 and Fabric2.
+    //
+    // Fabric1 has 2 sessions, and Fabric2 has 3 sessions. Fabric2 will be selected since
```

Comment:
```suggestion
    // Fabric1 has 2 sessions, and Fabric2 has 4 sessions. Fabric2 will be selected since
```

Reply:
Damn, you actually read the tests! Thanks for catching this.